code/modules/antagonists/heretic/items/forbidden_book.dm

```diff
@@ -39,8 +38,12 @@
 
 	if(book_open)
 		close_animation()
+		RemoveElement(/datum/element/heretic_focus)
+		w_class = WEIGHT_CLASS_SMALL
 	else
 		open_animation()
+		AddElement(/datum/element/heretic_focus)
+		w_class = WEIGHT_CLASS_NORMAL
```

Comment:
Also: using the codex to tap an influence also does the open animation, but doesn't make it a focus, which might be confusing for people